simple_ai_agents/Recruitify/agents.py

```
@@ -0,0 +1,675 @@
+            if hasattr(pdf_file, 'getvalue'): 
+                pdf_data = pdf_file.getvalue()
+                pdf_file_like = io.BytesIO(pdf_data)
+                reader = PyPDF2.PdfReader(pdf_file_like)
+            else: 
+                reader = PyPDF2.PdfReader(pdf_file)
+                
+            text = ""
+            for page in reader.pages:
+                text += page.extract_text()
+            
+            return text
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)